street_lights/index.html

```diff
@@ -1,12 +1,56 @@
-<!DOCTYPE html>
 <html>
   <head>
     <meta charset="utf-8">
     <title>Światła drogowe</title>
   </head>
   <body>
     <div id="streetLights">
-      <!-- put here the code for you streetLights -->
+      <div class="light red">
+      </div>
+      <div class="light yellow">
+      </div>
+      <div class="light green">
+      </div>
+    </div>  
+    <div id="streetLights">
+        <div class="light red off">
+        </div>
+        <div class="light yellow off">
+        </div>
+        <div class="light green off">
+        </div>
     </div>
+    <div id="streetLights">
```

Comment:
No dobra niech i tak bedzie, teraz dziala ale to jest blad semantyczny
nie mozesz uzywac id do wielu elementów. Zmień to na coś innego co bedzie mozna uzywac w tej sytuacji. Po co Ci to id ?